order/order_iso.lean

```diff
@@ -289,18 +289,6 @@ instance (r : α → α → Prop) [is_well_order α r]
   (p : set α) : is_well_order p (subrel r p) :=
 order_embedding.is_well_order (subrel.order_embedding r p)
 
-instance [preorder α] (p : set α) : preorder p :=
```

Comment:
We have `subtype.preorder`, but I don't think it matches with the `set A -> Type` coercion.
Is there a reason why you want to remove them?

Reply:
Well I sneaked these three instances in in #462 and things have been working fine long before them.